src/Compilers/CSharp/Portable/Symbols/Source/SourceAssemblySymbol.cs

```diff
@@ -824,6 +828,10 @@ private void DetectAttributeAndOptionConflicts(DiagnosticBag diagnostics)
                     }
                 }
             }
+            if (_compilation.Options.PublicSign && (object)this.AssemblyKeyFileAttributeSetting != (object)CommonAssemblyWellKnownAttributeData.StringMissingValue)
```

Comment:
Possibly. However, the method is structured with delay signing errors at the top, key container errors in the middle, and key file errors at the bottom. The two things I added are next to their associated sections - it'd be weird to yank the key container check to the bottom (and there wouldn't be much code sharing - pretty much only the PublicSign property access).

(Forgot empty line between braces since these were originally else-ifs - having the behavior of only reporting when there wasn't a commandline /keyfile. Removing the else makes it report in all cases when an attribute is present)